pkg/dashboard/core/common.go

```diff
@@ -62,6 +62,8 @@ func (f *Filter) toMap() map[string]interface{} {
 	return fMap
 }
 
+const zeroTime = "0001-01-01 00:00:00"
```

Comment:
do we need to use this to initialize `Start` and `End` in `Filter`?

Reply:
I think it's no need to initialize them in the `Filter` struct because if we don't pass start or end, the `time.Parse` will always return zero time.